apps/address-book/package.json

```diff
@@ -17,7 +17,7 @@
     "prepublishOnly": "truffle compile --all",
     "publish": "aragon apm publish patch && sleep 3",
     "publish:cd": "aragon apm publish major --files dist/ --environment continuous-deployment --apm.ipfs.rpc https://ipfs.autark.xyz:5001 --ipfs-check false",
```

Comment:
--skip-confirmation true seems to missing, I would ensure it is present on every `aragon apm publish`,  `aragon run` or `aragon start` call, to keep them working unattended